.gitignore

```diff
@@ -107,3 +107,17 @@ build/android/obj
 build/android/local.properties
 build/android/.gradle
 timestamp
+
+#My own ignore files
+"nbproject/configurations (Jarvis's conflicted copy 2018-06-13).xml"
```

Comment:
nbproject
src/cmake_config*

Reply:
Why is .gitignore even included? I can't get it not to be included in this PR...

Reply:
You've deleted the file which is why it's included. You need to readd it with no changes for it to disappear from this PR. You should also delete any nbproject things